images/PoweredbyJupyter-darkborder.svg

```diff
@@ -0,0 +1,124 @@
+                                                    </g>
+                                                </g>
+                                                <g id="path" transform="translate(113.098039, 5.490196)">
+                                                    <g id="path5-fill">
+                                                        <g id="path5_fill-link" transform="translate(0.823529, 0.779608)">
+                                                            <path d="M3.98423529,13.0977412 C4.08001725,18.8931922 7.71945725,21.2790118 11.9239373,21.2790118 C14.122102,21.3486275 16.3088471,20.9332392 18.3313255,20.0619451 L19.0496627,23.1239373 C16.6264,24.1502745 14.0151529,24.6473569 11.3876549,24.5824627 C4.26198431,24.5824627 0,19.8108235 0,12.7016784 C0,5.59262118 4.11832784,0 10.8608816,0 C18.4175216,0 20.4383529,6.76137412 20.4383529,11.0983216 C20.4312157,11.7671373 20.3833412,12.4348549 20.2947294,13.0977412 L4.01297098,13.0977412 L3.98423529,13.0977412 L3.98423529,13.0977412 Z M16.3296,10.0358149 C16.3774745,7.31194196 15.2282667,3.0716 10.4969365,3.0716 C6.23495216,3.0716 4.37691608,7.06080941 4.04170667,10.0358149 L16.3391529,10.0358149 L16.3296,10.0358149 L16.3296,10.0358149 Z" id="path5_fill"></path>
+                                                        </g>
+                                                    </g>
+                                                </g>
+                                                <g id="path" transform="translate(139.450980, 5.490196)">
+                                                    <g id="path6-fill">
+                                                        <g id="path6_fill-link" transform="translate(0.845490, 0.790588)">
+                                                            <path d="M0.191549647,7.88231843 C0.191549647,5.11015373 0.143661961,2.72435608 0,0.531737569 L3.68733647,0.531737569 L3.83099294,5.15844549 L4.02254588,5.15844549 C5.07607059,1.99026196 7.61411137,0.000486244706 10.4298902,0.000486244706 C10.8335843,-0.00546669804 11.2362353,0.0432640627 11.6270275,0.145372706 L11.6270275,4.18288157 C11.1553098,4.0791498 10.6732596,4.03053961 10.1904518,4.03799529 C7.22143059,4.03799529 5.12395608,6.32719843 4.54930824,9.53402196 C4.43076392,10.1940204 4.36669333,10.8627812 4.35775529,11.5334745 L4.35775529,24.0903216 L0.191549647,24.0903216 L0.191549647,7.88231843 L0.191549647,7.88231843 Z" id="path6_fill"></path>
+                                                        </g>
+                                                    </g>
+                                                </g>
+                                            </g>
+                                        </g>
+                                        <text id="POWERED-BY" font-family="MyriadPro-Bold, Myriad Pro" font-size="10" font-weight="bold" letter-spacing="2" fill="#757575">
+                                            <tspan x="8" y="8">POWERED BY</tspan>
```

Comment:
The brand guidelines say:

> If you have a [thing] that uses [Jupyter software] or relies on its architecture, we recommend […]

This makes clear that it applies to software that *uses* Jupyter components; e.g. PyCharm has a notebook frontend powered by Jupyter, and projects like [atom-notebook](https://github.com/jupyter/atom-notebook) and [hydrogen](https://atom.io/packages/hydrogen) use various parts of its architecture to run code and get output.

However, this doesn’t apply to us: Language kernels are the fundamental part of Jupyter’s architecture. They don’t use anything, they just provide a Jupyter protocol compliant interface to a programming language. (Nitpickers might say that’s a part of its architecture, but keep reading.)

The true relationship is that Jupyter software (can) use the IRkernel; All parts of Jupyter are in the end “powered by a kernel”.

I understand the need to distinguish projects that aren’t directly part of the jupyter project (Add-ons, so to speak), but “powered by” doesn’t fit for language kernel projects.